columnflow/production/cms/seeds.py

```diff
@@ -25,19 +25,23 @@
     uses={
         # global columns for event seed
         "run", "luminosityBlock", "event",
-        *optional({
-            "Pileup.nPU",
-            # columns needed to extract object counts only
-            "Photon.pt", "SV.pt", "FatJet.pt", "SubJet.pt",
-            "GenJet.pt", "GenPart.pt",
-            # per-object columns for further hashing
-            "Electron.jetIdx", "Electron.seediPhiOriY",
-            "Muon.jetIdx", "Muon.nStations",
-            "Tau.jetIdx", "Tau.decayMode",
-            "Jet.nConstituents", "Jet.nElectrons", "Jet.nMuons",
-        }),
+        # all other columns to be loaded are defined through the lists below
     },
     produces={"deterministic_seed"},
+    # columns of event integer fields
+    event_columns=[optional("Pileup.nPU")],
+    # columns to be loaded to infer counts of objects
+    object_count_columns=list(map(optional, [
+        "Jet.nConstituents", "FatJet.pt", "SubJet.pt", "Photon.pt", "Muon.jetIdx",
+        "Electron.jetIdx", "Tau.jetIdx", "SV.pt", "GenJet.pt", "GenPart.pt",
```

Comment:
Why don't we use one observable consistently, e.g. pt? Is this just for demonstration purposes or is there a deeper reason?

Reply:
Because, e.g., we already need `Tau.jetIdx` as an object column, and upon loading, we can infer the object counts from there, so there's no need to load an additional pt column per tau.